js/web/lib/wasm/jsep/webgpu/ops/attention.ts

```diff
@@ -282,12 +282,12 @@ const createInPlaceSoftmaxProgramInfo = (_context: ComputeContext, input: Tensor
     })()};
     workgroupBarrier();
 
-    var max_value: f32 = -3.402823e+38f;
+    var max_value = -3.402823e+38f;
     for (var i = 0u; i < ${WG}; i++) {
       max_value = max(thread_max[i], max_value);
     }
 
-    var sum_vector = ${inputHelper.type.value}(${0});
+    var sum_vector = ${f32Type}(${0});
```

Comment:
Nit: (0) instead of (${0})

Reply:
fair. let me merge this one and add this in the next pr so I don't need to go through ci